test/support/utils.h

```diff
@@ -930,6 +932,41 @@ get_new_kernel_params(KernelParams)
 }
 #endif //TEST_DPCPP_BACKEND_PRESENT
 
+template <typename T>
+typename ::std::enable_if_t<std::is_arithmetic_v<T>, void>
```

Comment:
```suggestion
typename ::std::enable_if_t<std::is_arithmetic_v<T>>
```

according to https://en.cppreference.com/w/cpp/types/enable_if

Reply:
Done